Add Singularity Guppy usage

hoelzer opened the pull request:
Hey, I added commands to run Guppy via Singularity on the RKI HPC. 

Can you test that as well? 

If it works, we have this part also complete. 

Please feel then free to merge directly.

hoelzer committed:
Add Singularity Guppy usage

Commit:
wip day02

hoelzer commented:
Ah damn, I submitted changes to day02 to the branch. Anyway, can be merged then along with the singularity changes ;)

matthuska commented on day01-tuesday/hands-on.md:
```diff
@@ -158,14 +158,36 @@ guppy_basecaller –i ./fast5 –s ./guppy_out –c dna_r10.4.1_e8.2_260bps_sup.
+
+# Start the Singularity image with GPU support
+singularity run --nv guppy_gpu-6.4.6-1--2c17584.img
+
```
# Make sure you are in the correct directory, which contains the fast5/ subdirectory
cd 2023-08-nanopore-workshop-example-bacteria
ls
# output:
# fast5  fastq  minknow_report.html  pycoqc.html  sequencing_summary_small.txt

matthuska committed:
Update hands-on.md

Add commands to change into the appropriate directory before running guppy itself.

matthuska commented:
Works for me, merging.
